lib/msprime.c

```diff
@@ -5888,6 +5884,10 @@ msp_activate_population(msp_t *self, int population_id)
         ret = MSP_ERR_POPULATION_PREVIOUSLY_ACTIVE;
         goto out;
     }
+    if (pop->state == MSP_POP_STATE_ACTIVE) {
```

Comment:
This isn't being covered by tests. Need to try to activate an already active population.